scripts/backup

```diff
@@ -36,6 +36,15 @@ then
     ynh_backup --src_path="/etc/php/${phpversion}/fpm/pool.d/$app.conf"
 fi
 
+#=================================================
+# BACKUP CUSTOM CONF
+#=================================================
+
+if [ $custom_error_file ]
```

Comment:
i'm very confused about this ... pretty sure this doesn't do what the author think this does ... for example if the variable is equal to "0" or "false": 

```shell
$ if [ 0 ]; then echo true; else echo false; fi
true
$ if [ false ]; then echo true; else echo false; fi
true
```

hence : 

```suggestion
if [[ "$custom_error_file" == 1 ]]
```

Reply:
It's actually redundant with a line above, isn't it? 

https://github.com/YunoHost-Apps/my_webapp_ynh/blob/25ec30c166324949c9deb3e0b01d7051386e665b/scripts/backup#L28